scripts/inner/kubestellar-kube-bind

```diff
@@ -18,6 +18,7 @@ space_name=""
 resource_to_bind=""
 start_konnector=false
 cm_namespace=kubestellar # the namespace where the configmaps live in on the provider side
+log_dir=$(pwd)/kubestellar-kube-bind-logs
```

Comment:
Is there any promise what is the current working directory at this point? Any promise that it is appropriate for putting logs into?